src/components/Product/ProductList/ProductContent/ProductRadioBox/ProductInfo/ProductInfo.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { ReactComponent as RigthArrow } from '../../../../../../assets/icon/arrow_right.svg';
+import './ProductInfo.scss';
+
+const ProductInfo = ({
+  spotdata: { isNew, title, description, price, url, alt },
+  isRadioActive,
+  handleInfoMouseEnter,
+  handleInfoMouseLeave,
+}) => {
+  return (
+    <Link
+      to={url}
+      alt={alt}
+      className={isRadioActive ? 'container active' : 'container'}
```

Comment:
공통적인 부분 제외하고 바뀌는 부분만 조건부로 적용할 수 있을 것 같습니다.

```suggestion
      className={`container${isRadioActive ? ' active' : ''}`}
```

Reply:
앗 세상에! 이런 방법이 있군요! 감사합니다!! 
수정완료했습니다! 앞으로 이렇게 클래스명을 변경하는 습관을 들이겠습니다 😊